services/subscriber/service.go

```diff
@@ -285,7 +293,8 @@ func (s *Service) updateSubs(wg *sync.WaitGroup) error {
 				}
 				sub, err := s.createSubscription(se, si.Mode, si.Destinations)
 				if err != nil {
-					return err
+					s.Logger.Println("createSubscription failed for:", si.Name)
```

Comment:
Returning the error will only silently stop processing the rest of the subscriptions. The new code will log the error and continue setting up the rest of the subscriptions.